### PR TITLE
Abandoned mods

### PR DESCRIPTION
**Pull request by cbc02009**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Further notes

* Create a folder and readme for mods abandoned by the original author.
* Move decontaminator and purge bucket mod to abandoned mods

### CODE REVIEW

**Comment by cbc02009** [comment]
Can't wait to see how many links this PR breaks.

File: abandoned_mods/printer_mods/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Abandoned Mods
+
+This is a list of mods that are no longer being maintained by the original author. Mods make it into this folder if the Dev Team feels they are popular or important enough to remain in the repository even though the original author is not longer updating the mod.
+
+For a mod to be included in this list, the following conditions need to be met:
+1. An issue or PR has been opened regarding the mod
+2. An attempt was made to contact the original author either on github or discord
+3. The autho has not responded for a period of at least two months
```
Comment (FHeilmann) [inline abandoned_mods/printer_mods/README.md:8]: typo

**Review by FHeilmann — COMMENTED** [review]
One small typo, otherwise lgtm.

**Comment by KiloQubit** [comment]
Other than the typo mentioned above, LGTM as well.

**Comment by cbc02009** [comment]
Typo fixed

**Review by KiloQubit — APPROVED** [review]

**Pull request merged**